package.json

```diff
@@ -6,6 +6,7 @@
   "main": "dist/reflex.js",
   "browser": "dist/reflex.js",
   "homepage": "https://obartra.github.io/reflex",
+  "files": ["package.json", "README.md", "LICENSE", "src", "dist"],
```

Comment:
do we want to include `src` in the published package? only `"dist"` should be needed unless you want it as reference